src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -257,250 +211,305 @@ class BasePaymentsPage extends React.Component {
-                            onCancel={this.hideDefaultDeleteMenu}
-                            contentStyles={!this.props.isSmallScreenWidth ? [styles.sidebarPopover, styles.willChangeTransform] : undefined}
-                            title={this.props.translate('paymentsPage.deleteAccount')}
-                            prompt={this.props.translate('paymentsPage.deleteConfirmation')}
-                            confirmText={this.props.translate('common.delete')}
-                            cancelText={this.props.translate('common.cancel')}
-                            anchorPosition={{
-                                top: this.state.anchorPositionTop,
-                                right: this.state.anchorPositionRight,
+                        )}
+                        {isPayPalMeSelected && (
+                            <Button
+                                onPress={navigateToAddPaypalRoute()}
+                                style={[styles.mb4]}
+                                text={translate('common.edit')}
+                                shouldUseDefaultHover
+                            />
+                        )}
+                        <Button
+                            onPress={() => {
+                                setShowConfirmDeleteContent(true);
                             }}
-                            shouldShowCancelButton
+                            style={[shouldShowMakeDefaultButton ? styles.mt4 : {}]}
+                            text={translate('common.delete')}
                             danger
                         />
-                    )}
-                </Popover>
-            </ScreenWrapper>
-        );
-    }
+                    </View>
+                ) : (
+                    <ConfirmContent
+                        onConfirm={() => {
+                            hideDefaultDeleteMenu(false);
+                            deletePaymentMethod();
+                        }}
+                        onCancel={hideDefaultDeleteMenu}
+                        contentStyles={!isSmallScreenWidth ? [styles.sidebarPopover, styles.willChangeTransform] : undefined}
+                        title={translate('paymentsPage.deleteAccount')}
+                        prompt={translate('paymentsPage.deleteConfirmation')}
+                        confirmText={translate('common.delete')}
+                        cancelText={translate('common.cancel')}
+                        anchorPosition={{
+                            top: anchorPosition.anchorPositionTop,
+                            right: anchorPosition.anchorPositionRight,
+                        }}
+                        shouldShowCancelButton
+                        danger
+                    />
+                )}
+            </Popover>
+        </ScreenWrapper>
+    );
 }
 
 BasePaymentsPage.propTypes = propTypes;
 BasePaymentsPage.defaultProps = defaultProps;
+BasePaymentsPage.displayName = BasePaymentsPage;
```

Comment:
There was a typo here. It should have been `'BasePaymentsPage'`.
And later it caused [this regression](https://github.com/Expensify/App/issues/28057).